src/core/services/interaction/interaction.js

```diff
@@ -0,0 +1,68 @@
+angular
+  .module('material.core.interaction', [])
+  .service('$mdInteraction', MdInteractionService);
+
+function MdInteractionService($timeout) {
```

Comment:
Regardless of whether we make the docs public, it is always nice to have them for ourselves :smile: